index.html

```diff
@@ -6,8 +6,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Tyom</title>
     <link rel="stylesheet" href="styles.css">
```

Comment:
Add styles.css after bootstrap and other libraries so that it overwrites styles from libraries